### ISSUE TITLE
oIoU or mIoU ?

### ISSUE THREAD

**Issue by qiulesun** [kkakkkka/ETRIS#4]
@kkakkkka 

Nice work !

As reported in paper, oIoU is the main metric. But, the released code computes the mIoU, which is similar to CRIS (also see [issue](https://github.com/DerrickWang005/CRIS.pytorch/issues/12)). So,  this inconsistency bothering me.

**Comment by kkakkkka** [comment]
Hi @qiulesun, I am awfully sorry for not replying in time. As our code is built upon the CRIS implementation, our response aligns with that of the CRIS author, which is **mIoU**.

**Comment by qiulesun** [comment]
@kkakkkka 

In terms of ablation study of Bridge in Table 6, the paper reports 67.73% IoU on RefCOCO val set. However, I get that results of 69.97% via only commenting these [lines](https://github.com/kkakkkka/ETRIS/blob/main/model/bridger.py#L106-L126) and other default settings keep unchanged, which is better than 67.73%. So, can you give some tips to reproduce 67.73% IoU on RefCOCO val set.

**Comment by kkakkkka** [comment]
Hello Qiule,

Thank you for your attention to the details of our project. I have also had discussions with other people who are interested in this project. I've noticed that the experimental results may vary across different machines and environments. Initially, I conducted the experiments using a 40G A100 GPU with Python 3.9 or 3.10. You might be able to reproduce our results by training on a machine with similar specifications.
